### DISCUSSION TITLE
Keybinding conflict: super+k triggers ctrl+l functionality in Ghostty 1.0

### DISCUSSION THREAD

**Discussion by ryansobol** [ghostty-org/ghostty#4114] [Feature Requests, Ideas] [answered]
I've encountered an unexpected behavior with custom keybindings in Ghostty 1.0 where `super+k` is triggering `ctrl+l` functionality.

Steps to reproduce:
1. Configure Ghostty with a `super+k=clear_screen` keybind (set by default)
2. Configure fish shell with `fzf_configure_bindings --git_log=\cl` binding `ctrl+l` to display fzf's git log (uses [fzf.fish](https://github.com/PatrickF1/fzf.fish))
3. Press `super+k`

Expected behavior:
- `super+k` should trigger the `clear_screen` action as configured in Ghostty

Actual behavior:
- `super+k` incorrectly triggers the `ctrl+l` binding (i.e. fzf's git log, in this case) instead of `clear_screen`

Environment:
- Ghostty 1.0.0
- Fish 3.7.1
- macOS 15.1.1
- fzf.fish 10.3

Here is an [asciinema recording](https://asciinema.org/a/tI4UUz7wzsTGJ14Be9WaNvC69) demonstrating the issue. My apologizes on the low contrast with comment syntax highlighting.

![demo](https://github.com/user-attachments/assets/81bff688-1421-4a9b-b223-3e4a980dd353)

**Comment by mitchellh** [comment, marked as answer]
Thanks for the issue. I'm going to close this since no one has been able to reproduce this and there doesn't seem to be a path to actionability here. This is also on an ancient version of Ghostty. If this still happens on tip (nightly) please open a new discussion.
